README.md

```diff
@@ -1,10 +1,9 @@
 [gem]: https://rubygems.org/gems/minitest-reporters
-[travis]: https://travis-ci.org/kern/minitest-reporters
 
 # minitest-reporters - create customizable Minitest output formats
 
 [![Gem Version](https://badge.fury.io/rb/minitest-reporters.svg)][gem]
-[![Build Status](https://secure.travis-ci.org/kern/minitest-reporters.png)][travis]
+[![Build Status](https://github.com/minitest-reporters/minitest-reporters/actions/workflows/ci.yml/badge.svg)][github-actions]
 [![Windows build status](https://ci.appveyor.com/api/projects/status/3pugsxatwcldgyjd/branch/master?svg=true)](https://ci.appveyor.com/project/os97673/minitest-reporters/branch/master)
```

Comment:
Worth also removing Appveyor section, since the last build was 3 years ago?

Reply:
I think it is worth to try to resurrect Appveyor (it has never been added to auto run because I didn't have permissions) now we have them)